spec/features/sort_review_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
```

Comment:
Trzeba to przenieść do reviews_spec i skorzystać z istniejącej konfiguracji. Można też użyć prostszej metody do weryfikacji porządku sortowania (tak jak pokazywałem na zajęciach).